h/accounts/test/views_test.py

```diff
@@ -47,6 +47,14 @@ def __init__(self, **kwargs):
             setattr(self, k, kwargs[k])
 
 
+class FakeSerializer(object):
+    def dumps(self, obj):
+        return 'faketoken'
+
+    def loads(self, token):
+        return {'username': 'foo@bar.com'}
```

Comment:
I think all these fake classes are fine, but they could also be done with the `mock` library, for example this one might be something like:

```
fakeSerializer = mock.Mock()
fakeSerializer.dumps.return_value = 'faketoken'
fakeSerializer.loads.return_value = {'username': 'foo@bar.com'}
```